ReceiptRecognizer/Program.cs

```diff
@@ -1,15 +1,92 @@
-﻿using Recognizer.ObjectRecognizer;
+﻿using DbEntities.Entities;
+using Recognizer.ObjectRecognizer;
 using Recognizer.TextRecognizer;
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
 
 namespace Recognizer
 {
-    class Program
+    public static class Program
     {
         static async Task Main(string[] args)
         {
             var recognizer = new ReceiptRecognizer(new EmguLargestAreaRecognizer(), new TesseractRecognizer());
-            await recognizer.RecognizeTestImages();
+            var image = Image.FromFile(@"C:\Users\aleks\Desktop\test\setOfImages\IMG_20201228_180236x.jpg");
```

Comment:
Are you using imge from your computer?

Reply:
Whole Program.cs file is used for testing purposes and is not accessed by web service.